features/old/accounts/accounts.feature

```diff
@@ -61,34 +62,37 @@ Feature: Account management
     And I follow "Settings"
     Then I should not see "API Key"
 
-  @wip @javascript
+  @javascript
   Scenario: For admins the account overview is a page to change account details
     And current domain is the admin domain of provider "foo.3scale.localhost"
     When I log in as provider "foo.3scale.localhost"
       And I go to the provider account page
-    Then I should see the page to change account details
+    Then I should see "Account Details"
```

Comment:
I'm not sure about this, the old step said `I should see the page to change account details`. For what I found, that step existed and was removed [here](https://github.com/3scale/porta/commit/8bcc7f77bf776619d2eea4a4bc1b1e6b0abfaec9#diff-81e56396f9ecae0f071257bcd9cfad5e4020e2a2a71cc7ed04aea290434bc005L65-L69). It seems there was a form in the Account Settings screen time ago, which was removed at some moment. After that, this cucumber doesn't make sense anymore. So now you converted this to a test to verify there's a link to the Account Details screen. I'm not sure how useful is that, but in any case, the scenario title should change to describe the new behavior.

Reply:
yes this can also be removed, as I was also not sure if removing was good idea or not so I fixed as much as I can.